app/routes/products_.$productId/route.tsx

```diff
@@ -11,6 +11,8 @@ import { isRouteErrorResponse, useLoaderData, useRouteError, json } from '@remix
 import { ProductNotFound } from '~/components/product-not-found/product-not-found';
 import { getUrlOriginWithPath } from '~/utils';
 import styles from './product-details.module.scss';
+import { Price } from '~/components/price/price';
+import { getProductPriceInfo } from '~/components/price/utils';
 
```

Comment:
sort imports

Reply:
maybe it makes sense to introduce some lint rule for this? cause there are so many possible ways to sort them

Reply:
sorted using VS code